src/components/Home.jsx

```diff
@@ -1,4 +1,7 @@
 import React from "react";
+import Container from 'react-bootstrap/Container';
```

Comment:
Can you please explain what changes did you do in Home.jsx?

Reply:
I've made some changes to Home.js, but it's still in progress. I haven't updated App.css yet.

Reply:
ah okss. will wait after you updated App.css. thanksssss

Reply:
Hi. after checking, looks like you already updated App.css. will approve this now since it looks good. nice job, Roy! :)